dev/clint/tests/rules/test_redundant_test_docstring.py

```diff
@@ -0,0 +1,179 @@
+from pathlib import Path
+
+from clint.config import Config
+from clint.linter import lint_file
+from clint.rules.redundant_test_docstring import RedundantTestDocstring
+
+
+def test_redundant_docstrings_are_flagged(index_path: Path) -> None:
+    code = '''import pytest
```

Comment:
```suggestion
    code = '''
```

Let's remove this. It's redudant.